files/preseeds/Centos.ERP.kickstart

```diff
@@ -25,11 +25,14 @@ network --hostname={{hostname}}
 services --enabled="chronyd"
 
 # Install to sda.
-bootloader --append=" crashkernel=auto" --location=mbr --boot-drive=sda
 ignoredisk --only-use=sda
-autopart --type=plain
 clearpart --all --initlabel --drives=sda
 
+part /boot/efi --fstype=efi --size=250
+part /boot --fstype=ext4 --size=512
+part / --fstype=ext4 --grow --size=10240
```

Comment:
Is this 10GB? I think we can safely make it 200GB at least

Reply:
10 GB is the minimum size, with "--grow" it will take up as much as it can.

Reply:
Ah! right!